README.md

```diff
@@ -1,22 +1,23 @@
 python-gp
-===========
+=========
 [![Python3.6](https://img.shields.io/badge/python-3.6-red.svg)](https://www.python.org/downloads/release/python-369/)
 [![Python3.7](https://img.shields.io/badge/python-3.7-red.svg)](https://www.python.org/)
 [![Python3.8](https://img.shields.io/badge/python-3.8-red.svg)](https://www.python.org/)
 [![GPL license](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-3.0.html)
 
 Cartesian Genetic Programming (CGP) in Python.
 
-This library implements Cartesian Genetic Programming for symbolic regression in pure Python. It provides Python data structures to represent and evolve a two-dimensional directed graph (the genotype) that can be translated into a computational graph (the phenotype) implementing a particular mathematical expression. The current implementation uses an evolutionary algorithm,  specifically (mu + lambda) evolution strategies, to evolve a population of symbolic expressions in order to optimize an objective function.
+This library implements Cartesian Genetic Programming (e.g, Miller and Thomson, 2000; Miller, 2011) for symbolic regression in pure Python, targeting applications with expensive fitness evaluations. It provides Python data structures to represent and evolve two-dimensional directed graphs (genotype) that are translated into computational graphs (phenotype) implementing mathematical expressions. The computational graphs can be compiled as a Python functions, SymPy expressions (Meurer et al., 2017) or PyTorch modules (Paszke et al., 2017). The current implementation uses an evolutionary algorithm, specifically (mu + lambda) evolution strategies adapted from Deb et al. (2002), to evolve a population of symbolic expressions in order to optimize an objective function.
+
```

Comment:
I would rephrase the last sentence a bit. Rather than "current implementation", perhaps say "The library currently implements an evolutionary algorithm, specifically (mu + lambda) evolution strategies adapted from Deb et al. (2002), to evolve a population of symbolic expressions in order to optimize an objective function."

Reply:
ok